MISRA.md

```diff
@@ -28,11 +28,12 @@ Deviations from the MISRA standard are listed below:
 ### Suppressed with Coverity Comments
 | Deviation | Category | Justification |
 | :-: | :-: | :-: |
-| Directive 4.6 | Advisory | Basic numerical types are used in platform implementations, since the POSIX platform functions and openssl functions used in the platform implementations take arguments of these types. |
+| Directive 4.6 | Advisory | Basic numerical types are used in platform implementations, since the POSIX platform functions and OpenSSL functions used in the platform implementations take arguments of these types. |
+| Rule 8.13 | Advisory | The object pointed by `pNetworkContext` is not modified by POSIX sockets or OpenSSL, but other implementations of `TransportRecv_t` and `TransportSend_t` may do so. |
```

Comment:
Let's move this to the "Flagged by Coverity". I tired putting your annotations in the header file and still no luck.